### DISCUSSION TITLE
Where is this parameter（pro） defined

### DISCUSSION THREAD

**Discussion by rgw123** [frangoteam/FUXA#792] [General]
![image](https://user-images.githubusercontent.com/112950683/231756034-ef13fa4c-454f-4865-8917-aefe09e59d5c.png)

**Comment by unocelli** [comment]
I agree that the code is not prefect. if you want to make changes to fuxa I think it would be better to try debugging, however look at who calls the method, I suppose is property of GaugeSettings that contains the tag id with which the control is bound.
